easybuild/easyconfigs/x/X11/X11-20180128-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,216 @@
+    ('intltool', '0.51.0', '-Perl-5.26.1'),
+]
+
+default_easyblock = 'ConfigureMake'
+
+common_specs = {
+    'sources': [SOURCE_TAR_GZ],
+    'start_dir': '%(name)s-%(version)s',
+}
+components = [
+    ('xorg-macros', '1.19.1', {                             # 2017-01-29
+        'sources': ['util-macros-%(version)s.tar.gz'],
+        'start_dir': 'util-macros-%(version)s',
+    }),
+    ('libpthread-stubs', '0.4', common_specs),              # 2017-03-14
+    ('bigreqsproto', '1.1.2', common_specs),                # 2012-03-23
+    ('compositeproto', '0.4.2', common_specs),              # 2010-10-30
+    ('damageproto', '1.2.1', common_specs),                 # 2010-10-30
+    ('dmxproto', '2.3.1', common_specs),                    # 2011-01-06
+    ('dri2proto', '2.8', common_specs),                     # 2012-07-11
+    ('dri3proto', '1.0', common_specs),                     # 2013-11-01
+    ('fixesproto', '5.0', common_specs),                    # 2011-03-08
+    ('fontsproto', '2.1.3', common_specs),                  # 2014-04-14
+    ('glproto', '1.4.17', common_specs),                    # 2013-12-10
+    ('inputproto', '2.3.2', common_specs),                  # 2016-04-04
+    ('kbproto', '1.0.7', common_specs),                     # 2015-05-01
+    ('presentproto', '1.1', common_specs),                  # 2017-01-26
+    ('randrproto', '1.5.0', common_specs),                  # 2015-05-17
+    ('recordproto', '1.14.2', common_specs),                # 2012-03-23
+    ('renderproto', '0.11.1', common_specs),                # 2010-08-10
+    ('resourceproto', '1.2.0', common_specs),               # 2011-05-28
+    ('scrnsaverproto', '1.2.2', common_specs),              # 2012-03-23
+    ('videoproto', '2.3.3', common_specs),                  # 2016-03-11
+    ('xcmiscproto', '1.2.2', common_specs),                 # 2012-03-23
+    ('xextproto', '7.3.0', common_specs),                   # 2013-12-27
+    ('xf86bigfontproto', '1.2.0', common_specs),            # 2009-08-27
+    ('xf86dgaproto', '2.1', common_specs),                  # 2009-10-01
+    ('xf86driproto', '2.1.1', common_specs),                # 2011-01-06
+    ('xf86vidmodeproto', '2.3.1', common_specs),            # 2011-01-06
+    ('xineramaproto', '1.2.1', common_specs),               # 2011-01-06
+
+    ('xproto', '7.0.31', common_specs),                     # 2016-09-23
+    ('libXau', '1.0.8', common_specs),                      # 2013-05-24
+    ('libXdmcp', '1.1.2', common_specs),                    # 2015-03-21
+    ('xcb-proto', '1.12', common_specs),                    # 2016-05-18
+    ('libxcb', '1.12', {                                    # 2016-05-18
+        'sources': [SOURCE_TAR_GZ],
+        'start_dir': '%(name)s-%(version)s',
+        'patches': ['libxcb_indent.patch'],
+
+    }),
+    ('xtrans', '1.3.5', common_specs),                      # 2014-09-22
+    ('libxkbcommon', '0.7.2', {                             # 2017-12-16
```

Comment:
`0.8.0` was released Dec 15th 2017 (0.7.2 dates from Aug'17), it seems like you bumped the date but not the version? :)